### ISSUE TITLE
Consider making editing tests test selection position too

### ISSUE THREAD

**Issue by ayg** [w3c/web-platform-tests#5342]
@masayuki-nakano @rniwa 

The selection tests actually compute an expected position for the selection for each test, they just don't actually test the browser.  I did this because I didn't review the failures and fix spec bugs, so some of the expected selection positions are probably wrong.  I also didn't think about exactly where to put the new selection in a lot of cases, e.g., whether in a text node or its parent, inside or outside tags, etc.

Is it worth having the editing tests test the selection position regardless?  Some of the results will be wrong, but it could still be helpful for regression testing.  It would be a good idea to add some fuzz to the expectations, like allowing [] instead of {} and so on.  (Or maybe requiring [] in all cases where it's possible, matching Chrome/WebKit.)

Another consideration is that this might considerably increase the number of expected failures for editing tests.  I don't know if that's a big deal.  @jgraham?

**Comment by ayg** [comment]
I wrote a patch for this and reviewed all the new failures in backcolor.html in Firefox Developer Edition.  I didn't think the results were particularly helpful, because in every single one, Firefox behaved more or less equivalently to the spec -- e.g., before vs. after inline start/end tags.  I think this would only be useful if we did some normalization on the selection so that it only caught selections that were entirely in the wrong place.  That will require some thought to implement, but I have ideas.
